### PR TITLE
Refined top level readme

### LINKED ISSUE #11855: Refine top level readme
The top level readme contains some outdated information such as a link to the 1.0 release. I will work on a refinement.

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed  #11855

Signed-off-by: Jonas Helming <jhelming@eclipsesource.com>

#### What it does
Refines content and links in the top level readme

#### How to test
Read the readme and follow links

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,28 +31,46 @@
...
+
+
 ## Website
 
-[Visit the Theia website](http://www.theia-ide.org) for more [documentation](http://www.theia-ide.org/doc).
+[Visit the Eclipse Theia website](http://www.theia-ide.org) for more information and [the Theia documentation](http://www.theia-ide.org/doc).
+
+## Releases
+
+- [All available releases](https://github.com/eclipse-theia/theia/releases) are available on Github including changelogs.
```
Comment (kittaakos) [inline README.md:48]: Please change: `Github` -> `GitHub`

File: README.md [hunk 2]
```diff
@@ -37,28 +31,46 @@
...
 
 See [our roadmap](https://github.com/eclipse-theia/theia/wiki/Eclipse-Theia-Roadmap) for an overview about the current project goals and the upcoming releases.
 
 ## Getting Started
-Here you can find guides and examples for common scenarios:
-- [Run Theia in Gitpod - a Theia-based IDE for GitHub](doc/Developing.md#run-the-browser-example-with-gitpod)
-- [Develop a Theia application - your own IDE](https://www.theia-ide.org/doc/Composing_Applications.html)
-- [Develop a Theia plugin - a VS Code like extension](https://www.theia-ide.org/doc/Authoring_Plugins.html)
+Here you can find guides and examples for common scenarios to adopt Theia:
+- [Get an overview about how to get started](https://theia-ide.org/#gettingstarted) on the Theia website
```
Comment (kittaakos) [inline README.md:67]: `Get an overview about` -> `Get an overview of`

**Review by kittaakos — COMMENTED** [review]
Looks great 👍

File: README.md [hunk 3]
```diff
@@ -7,27 +7,21 @@
...
-  Eclipse Theia is an extensible framework to develop full-fledged multi-language Cloud & Desktop IDE-like products with state-of-the-art web  technologies.
-
-  **Theia 1.0 has been released. Read the [announcement](https://dev.to/svenefftinge/theia-1-0-finally-a-good-browser-ide-3ok0).**
-
-</div>
-
-<div style='margin:0 auto;width:80%;'>
-
-![Theia](https://raw.githubusercontent.com/eclipse-theia/theia/master/doc/images/theia-screenshot.png)
+  Eclipse Theia is an extensible framework to develop full-fledged multi-language Cloud & Desktop IDEs and tools with  state-of-the-art web technologies.
```
Comment (vince-fugnitto) [inline README.md:19]: minor: unnecessary whitespace:

```suggestion
  Eclipse Theia is an extensible framework to develop full-fledged multi-language Cloud & Desktop IDEs and tools with state-of-the-art web technologies.
```

File: README.md [hunk 4]
```diff
@@ -37,28 +31,46 @@
 - [**License**](#license)
 - [**Trademark**](#trademark)
 
+
+<div style='margin:0 auto;width:60%;'>
```
Comment (vince-fugnitto) [inline README.md:35]: Did we mean to make it 20% smaller?

**Review by vince-fugnitto — COMMENTED** [review]
Looks much better!
I have some comments in addition to what Akos mentioned.

**Review by msujew — COMMENTED** [review]
I like the changes as they are, but I would propose some additional ones:

> * Get familiar with [the development workflow](doc/Developing.md), [Coding Guidelines](doc/coding-guidelines.md), [Code of Conduct](CODE_OF_CONDUCT.md) and [learn how to sign your work](CONTRIBUTING.md#sign-your-work)

Change this to:

> * Get familiar with [the development workflow](doc/Developing.md), [Coding Guidelines](doc/coding-guidelines.md), [Code of Conduct](CODE_OF_CONDUCT.md) and [sign the Eclipse contributor agreement](CONTRIBUTING.md#eclipse-contributor-agreement)

File: README.md (hunk 4 above)
Comment (JonasHelming) [reply README.md:35]: yes, it was very big from my POV

**Comment by JonasHelming** [comment]
all feedback should be adressed

**Review by msujew — APPROVED** [review]
Thanks, the change looks good to me 👍

**Review by kittaakos — APPROVED** [review]

**Review by vince-fugnitto — APPROVED** [review]
LGTM
